### PR TITLE
Deprecate plugincheck v1

### PR DESCRIPTION
**Pull request by academo**
Closes https://github.com/grafana/plugin-validator/issues/35

- Remove plugincheck v1 from readme
- Add ascii deprecated message

### CODE REVIEW

File: pkg/cmd/plugincheck/main.go [hunk 1]
```diff
@@ -13,7 +13,29 @@ import (
...
+ .d88888 .d88b. 88888b. 888d888 .d88b.  .d8888b 8888b. 888888 .d88b.  .d88888
+d88" 888d8P  Y8b888 "88b888P"  d8P  Y8bd88P"       "88b888   d8P  Y8bd88" 888
+888  88888888888888  888888    88888888888     .d888888888   88888888888  888
+Y88b 888Y8b.    888 d88P888    Y8b.    Y88b.   888  888Y88b. Y8b.    Y88b 888
+ "Y88888 "Y8888 88888P" 888     "Y8888  "Y8888P"Y888888 "Y888 "Y8888  "Y88888
+                888
+                888
+                888
+
+	**plugincheck V1 is deprecated and you should not use it.
```
Comment (leventebalogh) [inline pkg/cmd/plugincheck/main.go:33]: Nit: do we need a closing `**` as well to make it bold? (In case that was the intention)

**Review by leventebalogh — APPROVED** [review]
Nice! 👏 

Probably a dumb question, but all of our pipelines are using the V2 now, right?

File: pkg/cmd/plugincheck/main.go (hunk 1 above)
Comment (academo) [reply pkg/cmd/plugincheck/main.go:33]: Not really this is not supposed to be markdown, just a way to catch attention in the terminal

**Comment by academo** [comment]
> Nice! clap
> 
> Probably a dumb question, but all of our pipelines are using the V2 now, right?

The community plugin pipeline is using v2. I'll have to check if we have other plugins around using this in their github action and using v1.

**Pull request merged**